outbound.js

```diff
@@ -421,18 +423,78 @@ exports.stats = function () {
     return results;
 };
 
-function _next_uniq () {
```

Comment:
I see what you're saying.  I figured the likelihood of getting two identical occurrences of 10 character alphanumeric random strings with an identical arrival moment and an identical next_attempt value would be vanishingly small (and it provides a visual cue for continuity if the same file sticks around for a while).